lib/muffin_man/fulfillment_inbound/v20240320.rb

```diff
@@ -38,6 +38,15 @@ def get_shipment(inbound_plan_id, shipment_id)
         call_api
       end
 
+      def get_shipment_items(inbound_plan_id, shipment_id, page_size: nil, pagination_token: nil)
```

Comment:
Looks like Amazon calls this `ListShipmentItems` rather than `get_shipment_items`. Might make sense to switch "get" for "list" throughout here? Though if we don't typically follow that convention in Muffin Man, then this should be just fine